### DISCUSSION TITLE
🎨 MonthPicker design idea

### DISCUSSION THREAD

**Discussion by zarahzachz** [CMSgov/design-system#2238] [Ideas]
I've been checking out [MonthPicker](https://design.cms.gov/components/month-picker/?theme=core) for a ticket I'm working on and was wondering if we considered having only one button to select/deselect all months? 

As it stands, one of the buttons will always be useless; if no months are selected, the "Clear all" button does nothing, if all months are selected, the "Select all" button does nothing. I'm wondering if consolidating these two buttons would improve UX?

**Comment by pwolfert** [comment]
Would we change the text of the button dynamically? I think maybe when these were written there was an accessibility concern about buttons entering and leaving the a11y tree. Like it was maybe better to have the a11y tree not change.
